src/components/organisms/RadioModal/RadioModal.tsx

```diff
@@ -1,20 +1,34 @@
-import React from "react";
+import React, { useCallback } from "react";
+import { Button } from "react-bootstrap";
 import { PLAYA_VENUE_NAME } from "settings";
 import "./RadioModal.scss";
 
 export interface RadioModalPropsType {
   volume: number;
   setVolume: Function;
   title?: string;
+  onEnableHandler: () => void;
+  isRadioPlaying: boolean;
 }
 
 export const RadioModal: React.FunctionComponent<RadioModalPropsType> = ({
   volume,
   setVolume,
   title,
+  onEnableHandler,
+  isRadioPlaying,
 }) => {
-  return (
-    <div className="radio-modal-container">
+  const handleEnableButtonClick = useCallback(() => {
+    onEnableHandler();
+    setVolume(volume);
+  }, [onEnableHandler, setVolume, volume]);
+
+  const renderEnableRadioButton = () => (
+    <Button onClick={() => handleEnableButtonClick()}>Enable radio</Button>
+  );
+
+  const renderRadioBody = () => (
+    <>
       <div className="title-radio">{title ?? `${PLAYA_VENUE_NAME} Radio`}</div>
       <div className="text-radio">
         We recommend turning on the global burner radio station as you rove
```

Comment:
Also need to delete the next line, which I don't seem to be able to suggest for you.
```suggestion
        We recommend turning on the radio as you explore the map!
```